measures/measures-schema.yaml

```diff
@@ -197,3 +205,55 @@ definitions:
       - thoracicSurgery
       - urology
       - vascularSurgery
+
+  eligibilityOption:
+    type: object
+    properties:
+      submissionMethods: { $ref: #/definitions/qualityCodesSubmissionMethods }
+      sexCode:
+        enum: [M, F]
+      minAge:
+        type: integer
+      maxAge:
+        type: integer
+      diagnosisCodes: { $ref: #/definitions/arrayOfStringIdentifiers }
+      additionalDiagnosisCodes: { $ref: #/definitions/arrayOfStringIdentifiers }
+      procedureCodes:
+        type: array
+        items: { $ref: #/definitions/codeObject }
+    required: [submissionMethods]
+
+  performanceOption:
+    type: object
+    properties:
+      submissionMethods: { $ref: #/definitions/qualityCodesSubmissionMethods }
+      optionType:
+        enum: [performanceMet, performanceNotMet, performanceExclusion]
```

Comment:
I think at the moment, given the current version of the [single source spreadsheet](https://github.com/CMSgov/claims-to-quality-analyzer/blob/master/data/qpp_measures/2017_Claims_IndivMeasures_SingleSource.xlsx) or [here in google drive](https://docs.google.com/spreadsheets/d/1u_WA9RtIdLAIg5RLA-gDruBRpHpxuPxv1NpoWYZCsso/edit#gid=538916146) we would have to parse PDFs to differentiate betwixt these 2 options.

But I think @akegan asks for changes to be made to that spreadsheet (by the contractor? or PQMM?) based on the logic required to make that file programmatically useful.

But given pushback on changes, it might be more expedient to parse PDFs.

I'll add this comment thread as an outstanding question in the tech spec and implementation ticket.